src/workerd/jsg/buffersource.c++

```diff
@@ -117,6 +128,14 @@ BackingStore BufferSource::detach(Lock& js) {
   return kj::mv(backingStore);
 }
 
+BackingStore BufferSource::cloneBackingStore(Lock& js) {
+  auto backingStore =
+      kj::mv(JSG_REQUIRE_NONNULL(maybeBackingStore,
```

Comment:
Is the `kj::mv()` intended? It looks like this function will leave the BufferSource in an unusable state, but perhaps that's intended.

Either way, a good test case to add to buffersource-test.c++. ;)

Reply:
No, good catch. Cut'n'paste issue.